### PR TITLE
chore: cleanup v2

### PR DESCRIPTION
**Pull request by cool-develope**

### CODE REVIEW

File: pool.go [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
 	"sync"
 )
 
 type NodePool struct {
 	syncPool *sync.Pool
-
-	free  chan int
-	nodes []Node
-
-	poolId uint64
```
Comment (tac0turtle) [inline pool.go:14]: whats the reason for this removal?

**Review by tac0turtle — APPROVED** [review]

File: tree_test.go [hunk 2]
```diff
@@ -111,10 +111,10 @@ func testTreeBuild(t *testing.T, multiTree *MultiTree, opts *testutil.TreeBuildO
...
 			node := changeset.GetNode()
 
-			//var keyBz bytes.Buffer
-			//keyBz.Write([]byte(node.StoreKey))
-			//keyBz.Write(node.Key)
-			//key := keyBz.Bytes()
+			// var keyBz bytes.Buffer
+			// keyBz.Write([]byte(node.StoreKey))
+			// keyBz.Write(node.Key)
+			// key := keyBz.Bytes()
```
[the remark this answers is not in the document]
Comment (tac0turtle) [reply tree_test.go:117]: ping

**Comment by julienrbrt** [comment]
Should we merge this or close this?

**Comment by tac0turtle** [comment]
matt is doing a follow up in a another pr, we can close this

**Pull request closed without merging**